content/appendices/symbol-lookup-cheatsheet.md

```diff
@@ -52,6 +52,7 @@ Here is a more elaborate explanation of Pony's use of special characters: (a lin
 | `;`  | only used to separate expressions on the same line |
 | `=`  | (1) (destructive) assignment |
 |    | (2) in: use alias = packname |
+|    | (3) default arguments and default types |
```

Comment:
I think this should be two entries. Each should get you to what you need with a search:

So something like

`Supply default type for generics`
`Supply default argument for method`

Reply:
I'll take that, reversing the order